src/configToOptions.js

```diff
@@ -41,10 +42,13 @@ function configToOptions(config = {}) {
 
   function getH2xPlugins() {
     const plugins = [jsx, stripAttribute('xmlns'), removeComments, removeStyle]
-    if (config.icon) plugins.push(emSize)
     config.replaceAttrValues.forEach(([oldValue, newValue]) => {
       plugins.push(replaceAttrValue(oldValue, newValue))
     })
+    if (!config.dimensions) {
+      plugins.push(...['width', 'height'].map(stripAttribute))
```

Comment:
@lifeiscontent 

@neoziro addressed this in my PR but this will remove all `height` and `width` attributes. Regardless if its in an `svg` or not. 

I think the best way is to actually implement a `h2x` plugin similar to [emSize](https://github.com/smooth-code/svgr/blob/master/src/h2x/emSize.js) 

Something like:

```js
const removeDimensions = () => ({
  visitor: {
    JSXElement: {
      enter(path) {
        // Skip if not svg node
        if (path.node.name !== 'svg') return

        const otherAttributes = []
        path.node.attributes.forEach(attr => {
          if (attr.name === 'width' || attr.name === 'height'){
          // NO OP
          } 
          else otherAttributes.push(attr)
        })

        path.node.attributes = otherAttributes
      },
    },
  },
})
```

I've tested this and it works :)